client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -73,7 +96,14 @@ class PostRelativeTime extends React.PureComponent {
 			statusText = this.props.translate( 'pending review' );
 			statusClassName += ' is-pending';
 		} else if ( status === 'future' ) {
-			statusText = this.props.translate( 'scheduled' );
+			const scheduledTime = this.props.moment( this.props.post.date ).fromNow();
```

Comment:
@Aurorum It should be in the stats now, so the fix should look like this (I haven't tested either):


```suggestion

                         const moment = this.props.moment;
                         const scheduledDate = moment(scheduledDate);
                         const now  = moment.now();
			if (now.diff( scheduledDate, 'days' ) >= 7) {
			      statusText = this.props.translate( 'scheduled for %(scheduledDate)s', {
			                   comment: '%(scheduledTime)s is a future human date with weekday, for example "Wed, 21 Apr 2020 14:00',
				           args: {
					         scheduledDate: scheduledDate.format( 'llll' ),
				            },
			                  } );
			  } else {
			             statusText = this.props.translate( 'scheduled for %(scheduledDate)s', {
			                   comment: '%(scheduledTime)s is a future human time, for example "in 3 days",
				           args: {
					         scheduledTime: scheduledDate.fromNow(),
				            },
			                  } );
			  }
			
```



So it would look like:
| < a week | > a week | 
| ------------- | ------------- |
| ![Screenshot 2020-04-06 at 17 51 05](https://user-images.githubusercontent.com/790558/78578050-3ec33d00-782f-11ea-99c4-efeac5c93aa4.png) | ![Screenshot 2020-04-06 at 17 54 55](https://user-images.githubusercontent.com/790558/78578399-c5781a00-782f-11ea-9dc8-ab617f8887f5.png)